README.md

```diff
@@ -50,27 +50,58 @@ Update configurations for individual projects in **`installer/ansible/group_var/
 
 Please refer [SODA documentation](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/) for the detailed installation steps and configuration options available.
 
+**Example#1:** To install Strato(Multi-cloud) and Dashboard using ansible installer.
 
 
-**Example:** To install Delfin, SRM tool chain and Dashboard using ansible installer.
+```bash
+sudo apt-get update && sudo apt-get install -y git
+git clone https://github.com/sodafoundation/installer.git
+# git checkout <installer-release-version>
```

Comment:
please remove #.
It should be part of steps:
git checkout 1.8.0  # you may choose specific release version also

Reply:
done